classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -683,7 +683,7 @@ protected function display_filter_by_status() {
 			}
 
 			if ( $status_name === $request_status || ( empty( $request_status ) && 'all' === $status_name ) ) {
-				$status_list_item = '<li class="%1$s"><strong>%3$s</strong> (%4$d)</li>';
+				$status_list_item = '<li class="%1$s"><a href="%2$s"><strong>%3$s</strong></a> (%4$d)</li>';
```

Comment:
To more closely match other WP Admin List Tables, I thint it makes sense to:

- Add class `current` to the `<a>` tag
- Drop the use of `<strong>` (since the above class should take care of formatting)

So:

```suggestion
				$status_list_item = '<li class="%1$s"><a href="%2$s" class="current">%3$s</a> (%4$d)</li>';
```

This should give us output that will render something like the following:

<img width="406" alt="subsubsub-links-current-screen-example" src="https://user-images.githubusercontent.com/3594411/172697232-4ec741d2-39fc-4063-b9b3-6706d9a89f19.png">

What do you think?

Reply:
Agreed. 

## Tests:
Manual tests done to see the effect. The other links adopt the class on being clicked like the all.

## Screenshot
![Screenshot 2022-06-14 at 09 59 55](https://user-images.githubusercontent.com/7713923/173514104-e2d1eb1a-7185-43a9-83df-71a28777f9d1.png)